lib/keyboard_utils.coffee

```diff
@@ -60,6 +62,7 @@ KeyboardUtils =
       if (event.keyCode >= @keyCodes.f1 && event.keyCode <= @keyCodes.f12)
         return "f" + (1 + event.keyCode - keyCodes.f1)
       return ""
+    return "backspace" if event.keyIdentifier == "U+0008"
```

Comment:
So?  Why can it **only** be searched in those circumstances?

(I'm not a fan of hard wiring key names into the logic.  We do it for the function keys, but only because there is a range of them.)